### ISSUE TITLE
Lack of automated tests 

### ISSUE THREAD

**Issue by leofdgit** [donadigo/gbxtools#3]
Typically, automated tests (unit, integration and system) are used to verify that code functions as expected. This repository does not contain a suite of automated tests (nor does the pygbx library, based on a quick glance). How have you verified that the code acts correctly? Further, is there any documentation detailing what the expected outputs of each script should be?

**Comment by uhdcc** [comment]
yes, i was just wondering this. I will try to contact donadigo and get a reply

**Comment by thecodix** [comment]
Hey, I'm willing to help with this
